contribute.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "about:home Snippets Service",
+    "description": "The Snippets Service hosts small chunks of HTML, CSS, and JavaScript that are displayed on about:home in Firefox, among other places.",
+    "repository": {
+        "url": "https://github.com/mozilla/snippets-service/",
+        "license": "BSD 3-Clause",
```

Comment:
Playdoh is BSD 3-Clause, thus most projects that were based on it are too. As to why Playdoh was BSD 3-Clause, I'm not really sure.